cmd/kubeops/main_test.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/kubeapps/kubeapps/cmd/kubeops/internal/handler"
+)
+
+func TestParseAdditionalClusterConfig(t *testing.T) {
+	testCases := []struct {
+		name           string
+		configJSON     string
+		expectedErr    bool
+		expectedConfig map[string]handler.AdditionalClusterConfig
+	}{
+		{
+			name:       "parses a single additional cluster",
+			configJSON: `[{"name": "cluster-2", "apiServiceURL": "https://example.com", "certificateAuthorityData": "abcd"}]`,
+			expectedConfig: map[string]handler.AdditionalClusterConfig{
+				"cluster-2": handler.AdditionalClusterConfig{
+					Name:                     "cluster-2",
+					ApiServiceURL:            "https://example.com",
+					CertificateAuthorityData: "abcd",
+				},
+			},
+		},
+		{
+			name: "parses multiple additional clusters",
+			configJSON: `[
+	{"name": "cluster-2", "apiServiceURL": "https://example.com/cluster-2", "certificateAuthorityData": "abcd"},
+	{"name": "cluster-3", "apiServiceURL": "https://example.com/cluster-3", "certificateAuthorityData": "efgh"}
+]`,
```

Comment:
wondering if it would make more sense to have this as a YAML to be more similar to a `.kube/config` but it's not the same so I guess it's okay as JSON too

Reply:
Yeah, I was unsure about this but went with json just because of the stdlib support. I don't think it'll be hard to configure this in the chart, but I'll find out next.